brickschema/web.py

```diff
@@ -44,6 +45,11 @@ def query(self):
     def home(self):
         return pkgutil.get_data(__name__, "web/index.html").decode()
 
+    def bindings(self):
+        ignored_prefixes = []
+        return jsonify(
+            {prefix: namespace for prefix, namespace in self.graph.namespaces() if prefix not in ignored_prefixes})
+
```

Comment:
I'm not sure I understood your comment on the PR; did you have an idea for how the `ignored_prefixes` would be populated?

Reply:
My bad, I didn't mean it to be static. I do think we should have a default list of `ignored_prefixes` tho (some prefixes that Brick would add that are rarely used in queries - if you think there are any, of course). 

Also, I prefer this one:
```python
graph.serve(ignored_namespaces=["bsh", "xsd"])
```
I think I should I rename that to `ignore_prefixes`.

Reply:
Okay, I've set it to be an empty list by default. This is how it will work now:

```python
graph.serve(ignore_prefixes=["bsh", "xsd"])
```

Reply:
Yes, I like this change (my second example above should have been `ignore_prefixes`, as you noted)